components/views/save.js

```diff
@@ -1,13 +1,61 @@
 import React from "react";
-import { Text, View } from "react-native";
-import { Standard } from "components/styles/save-style";
+import { Text, View, Image, FlatList } from "react-native";
+import Database from "../database";
+import saveStyle from "../styles/save-style";
 
-function SaveScreen() {
-  return (
-    <View style={Standard.container}>
-      <Text>Save!</Text>
+const renderSaveScreen = ({ item }) => (
+  <View style={saveStyle.card}>
+    <Text>{item.name}</Text>
+    <Image
+      source={{ uri: item.images[0] }}
+      resizeMode="cover"
+      overflow="hidden"
+      style={saveStyle.image}
+    />
+    <View style={saveStyle.cardContent}>
+      <Text>{item.name}</Text>
     </View>
-  );
-}
+  </View>
+);
+
+export default class SavedScreen extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      SaveData: [],
```

Comment:
```suggestion
      savedData: [],
```
seems you made little typo